src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -778,7 +767,6 @@ export const CURVE_FANTOM_POOLS = {
     twoPool: '0x27e611fd27b276acbd5ffd632e5eaebec9761e40',
     ren: '0x3ef6a01a0f81d6046290f3e2a8c5b843e738e604',
     tri_v2: '0x2dd7c9371965472e5a5fd28fbe165007c61439e1',
-    geist: '0x0fa949783947bf6c1b171db13aeacbb488845b3f',
```

Comment:
idk if i'd remove this pool from curve. we just want to offboard geist right? im seeing 11M$ in liquidity on curve https://ftm.curve.fi/geist (2nd largest pool)